### DISCUSSION TITLE
I got approved for GitHub education yesterday, but it still isn't given me access.

### DISCUSSION THREAD

**Discussion by alikirat** [community/community#139764] [GitHub Education]
### Hi everyone,

Hi all,
I got accepted for the GitHub education yesterday, but have yet to get access. How long did it take you for access. Isn't there any way to reach out to them to expedite it. I'm on a time crunch and need the access asap. 

Thanks!

**Comment by davevad93** [comment]
Hi @peripateticlearner ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @peripateticlearner was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by alikirat** [reply]
Thanks @davevad93 for your quick response. I guess it's just a waiting game. Hopefully, it gets accepted sooner than later.

**Reply by davevad93** [reply]
You're welcome @peripateticlearner , yes most users get their benefits in 3 days, however if yours takes longer don't worry because in the worst case scenario you'll have them in 2 weeks.
